<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UVAddDialog</class>
 <widget class="QDialog" name="UVAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="insertUV">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="test">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="code">
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="codeLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="titre">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titreLine">
        <property name="maxLength">
         <number>32766</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nombre de crédits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nbCreditspinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="saison">
        <property name="text">
         <string>Saison</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="saisonComb"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="categorie">
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categorieComb"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="tshTypeCombo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tshType">
    <item>
     <widget class="QLabel" name="tshLabelType">
      <property name="text">
       <string>Type TSH</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>391</width>
     <height>70</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="isFinalTest">
           <property name="text">
            <string>Final</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutFinalCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="isMdianTest">
           <property name="text">
            <string>Médian</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutMedianCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="isProjectTest">
           <property name="text">
            <string>Projet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutProjCombo"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="tauxDeReussite">
         <property name="text">
          <string>Taux de réussite récent </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="probaReussiteUV">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="isTP">
           <property name="text">
            <string>TP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutTpComb"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="isTD">
           <property name="text">
            <string>TD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="statutTDCombo"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>insertUV</zorder>
  <zorder>test</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>isFinalTest</zorder>
  <zorder>isMdianTest</zorder>
  <zorder>isProjectTest</zorder>
  <zorder>isTP</zorder>
  <zorder>isTD</zorder>
  <zorder>statutFinalCombo</zorder>
  <zorder>statutMedianCombo</zorder>
  <zorder>statutProjCombo</zorder>
  <zorder>statutTpComb</zorder>
  <zorder>statutTDCombo</zorder>
  <zorder>tauxDeReussite</zorder>
  <zorder>probaReussiteUV</zorder>
  <zorder></zorder>
  <zorder>statutProjCombo</zorder>
  <zorder>isProjectTest</zorder>
  <zorder>titreLine</zorder>
  <zorder>titre</zorder>
  <zorder>tshTypeCombo</zorder>
  <zorder>tshTypeCombo</zorder>
  <zorder></zorder>
  <zorder>label</zorder>
  <zorder>description</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
